string[] link
---
bool success
string message
